README.md

```diff
@@ -54,18 +54,113 @@ A smart contract project is a directory containing all the relevant contract and
 ### Package Open Project
 To package a project you have open in your workspace, run the `Package Open Project` command. Packages are listed in the `Smart Contracts` panel. The `Blockchain` output channel lists what files have been packaged during this action. Alternatively run the `Import a Package` command to import a pre-existing .cds package to be used within VS Code. 
 
-### Operate the Local Fabric runtime
-The extension contains a pre-configured local instance of Hyperledger Fabric named `Local Fabric`, which the extension will automatically pull and use the correct Docker images for. It is a pre-configured network with one organization, one peer and one channel. It can be enabled and operated under the `Local Fabric Ops` panel. The first time it is started, Fabric images will be installed and an admin identity created in the `Local Fabric Wallet` wallet. 
+### Connecting to an instance of Hyperledger Fabric
+#### Local Fabric
+The extension contains a pre-configured local instance of Hyperledger Fabric named `Local Fabric`, which the extension will automatically pull and use the correct Docker images for. It is a pre-configured network with one organization, one peer and one channel. It can be enabled and operated under the `Fabric Environments` panel. The first time it is started, Fabric images will be installed and an admin identity created in the `local_fabric_wallet` wallet. 
```

Comment:
Not sure if this branch has the updated names, but this might need to be changed to 'Local Fabric Wallet'